src/unit_model.js

```diff
@@ -0,0 +1,99 @@
+import { InMemoryEntity } from "@exabyte-io/code.js/dist/entity";
+import { getUUID } from "@exabyte-io/code.js/dist/utils";
```

Comment:
We need to reuse this and other flowchart/graph-related logic in object-oriented way